assets/css/main.css

```diff
@@ -161,6 +161,15 @@ h6::before { color: var(--maincolor); content: '###### '; }
   letter-spacing: -0.5px;
 }
 
+.toc {
+  background-color: #272822;
```

Comment:
Won't this show the same color in light theme too? Wouldn't that kinda look awkward in the light theme

Reply:
Indeed, I'm a terrible designer. Give me some time for a more complete proposal.